vars/scyllaCloudLongevityPipeline.groovy

```diff
@@ -0,0 +1,113 @@
+#!groovy
+
+def call(Map pipelineParams) {
+    pipeline {
+        agent {
+            label {
+                label "aws-eu-west1-qa-builder1"
+            }
+        }
+        parameters {
+            string(defaultValue: 'i3.large',
+                   description: 'any type support by scylla cloud',
+                   name: 'db_instance_type')
+
+            string(defaultValue: "${pipelineParams.get('n_db_nodes', '3')}",
+                   description: 'any type support by scylla cloud',
+                   name: 'n_db_nodes')
+
+            string(defaultValue: "${pipelineParams.get('provision_type', 'spot_low_price')}",
+                   description: 'spot_low_price|on_demand|spot_fleet|spot_low_price|spot_duration',
+                   name: 'provision_type')
+
+            string(defaultValue: "${pipelineParams.get('post_behaviour', 'destroy')}",
+                   description: 'keep|destroy',
```

Comment:
add a comment: for loader nodes

Reply:
no needed, now that this param is also used to destroy the scylla could